src/omnicore_rpctx.cpp

```diff
@@ -0,0 +1,873 @@
+            "PropertyIDForSale   : the property to list for sale (must be MSC or TMSC)\n"
+            "AmountForSale       : the amount to list for sale\n"
+            "AmountDesired       : the amount of BTC desired\n"
+            "PaymentWindow       : the time limit a buyer has to pay following a successful accept\n"
+            "MinAcceptFee        : the mining fee a buyer has to pay to accept\n"
+            "Action              : the action to take: (1) new, (2) update, (3) cancel \n"
+            "Result:\n"
+            "txid    (string) The transaction ID of the sent transaction\n"
+            "\nExamples:\n"
+            ">omnicored senddexsell_OMNI \"1FromAddress\" PropertyIDForSale \"AmountForSale\" \"AmountDesired\" PaymentWindow MinAcceptFee Action\n"
+        );
+
+    // obtain parameters & info
+    std::string fromAddress = params[0].get_str();
+    unsigned int propertyIdForSale = int64Touint32Safe(params[1].get_int64());
+    string strAmountForSale = params[2].get_str();
+    string strAmountDesired = params[3].get_str();
+    int64_t paymentWindow = params[4].get_int64();
+    int64_t minAcceptFee = StrToInt64(params[5].get_str(), true); // BTC so always divisible
+    int64_t action = params[6].get_int64();
+    const int64_t senderBalance = getMPbalance(fromAddress, propertyIdForSale, BALANCE);
+    const int64_t senderAvailableBalance = getUserAvailableMPbalance(fromAddress, propertyIdForSale);
+
+    // perform conversions
+    int64_t amountForSale = 0, amountDesired = 0;
+    if ((propertyIdForSale > 2 || propertyIdForSale <=0)) throw JSONRPCError(RPC_INVALID_PARAMETER, "Invalid propertyID for sale - only 1 and 2 are permitted");
+    amountForSale = StrToInt64(strAmountForSale, true); // TMSC/MSC always divisible
+    amountDesired = StrToInt64(strAmountDesired, true); // BTC so always divisible
+
+    // perform checks
+    if (action <= 0 || 3 < action) throw JSONRPCError(RPC_TYPE_ERROR, "Invalid action (1,2,3 only)");
+    if (action <= 2) { // actions 3 permit zero values, skip check
+        if (0 >= amountForSale) throw JSONRPCError(RPC_TYPE_ERROR, "Invalid amount for sale");
+        if (!isRangeOK(amountForSale)) throw JSONRPCError(RPC_TYPE_ERROR, "Amount for sale not in range");
+        if (0 >= amountDesired) throw JSONRPCError(RPC_TYPE_ERROR, "Invalid amount desired");
+        if (!isRangeOK(amountDesired)) throw JSONRPCError(RPC_TYPE_ERROR, "Amount desired not in range");
+    }
+    if (action != 3) { // only check for sufficient balance for new/update sell offers
+        if (senderBalance < amountForSale) throw JSONRPCError(RPC_TYPE_ERROR, "Sender has insufficient balance");
+        if (senderAvailableBalance < amountForSale) throw JSONRPCError(RPC_TYPE_ERROR, "Sender has insufficient balance (due to pending transactions)");
+    }
+    if (minAcceptFee <= 0) throw JSONRPCError(RPC_TYPE_ERROR, "Mininmum accept mining fee invalid");
```

Comment:
It's not validated, as it seems (regtest mode):
```
> sendrawtx_MP "mz2kMTWoHnuvapPkHRgCPQt5tZJNFjNwj7" "00010014000000010000000008f0d180000000174876e8000a000000000000000001"
82a313b0de75058054307ff092175772352d2dcae3ad71f98b393a3a80ee65f7

> setgenerate true 1
[
  "6eea2284a15a9d005a320405180c53325470c71e809525e0b7f9dad22acecc08"
]

> gettransaction_MP 82 a313b0de75058054307ff092175772352d2dcae3ad71f98b393a3a80ee65f7
{
  "txid": "82a313b0de75058054307ff092175772352d2dcae3ad71f98b393a3a80ee65f7",
  "sendingaddress": "mz2kMTWoHnuvapPkHRgCPQt5tZJNFjNwj7",
  "ismine": true,
  "confirmations": 1,
  "fee": 0.00009999,
  "blocktime": 1429930699,
  "version": 0,
  "type_int": 20,
  "type": "DEx Sell Offer",
  "propertyid": 1,
  "divisible": true,
  "amount": "1.50000000",
  "feerequired": 0.00000000, <<<
  "timelimit": 10,
  "subaction": "New",
  "bitcoindesired": 1000.00000000,
  "valid": true <<<
}
```

So let's tackle it this way: do you think it would make sense, to enforce a minimum on the RPC layer? I'm not sure actually, and I'm wondering, what benefit it provides, if zero commitment fee are not disallowed.